ckanext/ckanext-apply_permissions_for_service/ckanext/apply_permissions_for_service/templates/apply_permissions_for_service/new.html

```diff
@@ -139,7 +139,9 @@ <h3>{{ _('Details of the service and API you are applying to access') }}</h3>
 
     <hr>
   {% endif %}
-  <input class="btn btn-primary" type="submit" value="{{ _('Send access application') }}"></input>
-  <a class="btn btn-secondary">{{ _('Cancel') }}</a>
+  {% block application_form_actions %}
```

Comment:
On line 44 which starts with "_('Use this form to apply for a permission to use a service that is available", there is a sentence in the design that is missing in the Finnish translation, namely "Voit myös hakea lupaa välitoimijana toisen organisaation puolesta".  In addition, line 106, starting with "{{ form.select('subsystemCode'", which a form field for subsystem code, there is an extra label in the design, which says "Valitse organisaatiosi alijärjestelmä, jolle käyttölupa halutaan" - currently it seems to be missing. Otherwise seems ok to me, except that some things present in the design, namely after the "Date when access needed" field in the form, but I suppose these will be implemented in some other coming user stories later?

Reply:
I added new option for additional file in settings-page when apply-permission-form is set to be filled in api-catalog. Also tried to check about translations in transifex